lib/matplotlib/quiver.py

```diff
@@ -114,26 +122,60 @@
 scale : float, optional
     Scales the length of the arrow inversely.
 
-    Number of data units per arrow length unit, e.g., m/s per plot width; a
-    smaller scale parameter makes the arrow longer. Default is *None*.
+    Number of data values represented by one unit of arrow length on the plot.
+    For example, if the data represents velocity in meters per second (m/s), the
+    scale parameter determines how many meters per second correspond to one unit of
+    arrow length relative to the width of the plot.
+    Smaller scale parameter makes the arrow longer.
+
+    If *None*, a autoscaling algorithm is used to scale the arrow lenght to a
+    reasonable size, which is based on the average vector length and the number of
+    vectors.If the autoscaling does not yield a reasonable result, the user has to
+    manually define a scale.
```

Comment:
```suggestion
    By default, an autoscaling algorithm is used to scale the arrow lenght to a
    reasonable size, which is based on the average vector length and the number of
    vectors.
```
- If `None` is only a "not given" placeholder, we don't explicitly mention the value.
- The last sentence is implied for any optional value: "Specify an explicit value if the default is not suitable".
  No need to mention it explicitly.
  sp